README.it.md

```diff
@@ -0,0 +1,183 @@
+<br>
+
+<div align="center">
+
+[<img src="./resources/icon.png" width="144"/>](https://hydralauncher.site)
+
+  <h1 align="center">Hydra Launcher</h1>
+  
+  <p align="center">
+    <strong>Hydra è un game launcher con il proprio client bittorrent e autogestore di repacks.</strong>
+  </p>
+
+[![build](https://img.shields.io/github/actions/workflow/status/hydralauncher/hydra/build.yml)](https://github.com/hydralauncher/hydra/actions)
+[![release](https://img.shields.io/github/package-json/v/hydralauncher/hydra)](https://github.com/hydralauncher/hydra/releases)
+
+[![be](https://img.shields.io/badge/lang-be-orange)](README.be.md)
+[![pl](https://img.shields.io/badge/lang-pl-white)](README.pl.md)
+[![pt-BR](https://img.shields.io/badge/lang-pt--BR-green.svg)](README.pt-BR.md)
+[![ru](https://img.shields.io/badge/lang-ru-yellow.svg)](README.ru.md)
+[![uk-UA](https://img.shields.io/badge/lang-uk--UA-blue)](README.uk-UA.md)
+[![es](https://img.shields.io/badge/lang-es-red)](README.es.md)
+[![ita](https://img.shields.io/badge/lang-it-red)](README.it.md)
+
+![Hydra Catalogue](./docs/screenshot.png)
+
+</div>
+
+## Table of Contents
+
+- [A proposito](#a-proposito)
+- [Caratteristiche](#caratteristiche)
+- [Installazione](#installazione)
+- [Contribuire](#contribuire)
+  - [Uscisciti su Telegram](#unisciti-su-telegram)
+  - [Forka e Clona la tua repository](#forka-e-clona-la-tua-repository)
+  - [Modi in cui contribuire](#modi-in-cui-contribuire)
+  - [Struttura del Progetto](#struttura-del-progetto)
+- [Compila il codice sorgente](#compila-il-codice-sorgente)
+  - [Installa Node.js](#installa-nodejs)
+  - [Installa Yarn](#installa-yarn)
+  - [Installa le Node Dependencies](#installa-node-dependencies)
+  - [Installa Python 3.9](#installa-python-39)
+  - [Installa le Python Dependencies](#installa-python-dependencies)
+- [Variabili d'Ambiente](#variabili-di-ambiente)
+- [Esecuzione](#esecuzione)
+- [Compilazione](#compilazione)
+  - [Compilare il client bittorrent](#compilazione-client-bittorrent)
+  - [Compilare l'applicazione Electron](#compilazione-applicazione-electron)
+- [Collaboratori](#collaboratori)
+
+## A proposito
+
+**Hydra** è un **Game Launcher** con il proprio **Client BitTorrent** e **autogestore di repack**.
+<br>
+Il launcher è scritto in TypeScript (Electron) and Python, che gestisce il sistema di torrenting appoggiandosi a libtorrent.
+
+## Caratteristiche
+
+- Motore di ricerca automatizzato sulle fonti di repack dal [Megathread]("https://www.reddit.com/r/Piracy/wiki/megathread/")
```

Comment:
this part was not in the original readme, please keep only what already exists

**do not use external and piracy related links**